.github/pull_request_template.md

```diff
@@ -9,16 +9,49 @@
 
 ## Issue type checklist
 <!--- What type of change are you submitting? Check the boxes that apply: -->
-- [ ] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
-- [ ] **New feature**: non-breaking change that adds functionality.
-- [ ] **Breaking change**: fix or feature that will cause current functionality to change.
+
+- [] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
+- [] **New feature**: non-breaking change that adds functionality.
+- [] **Breaking change**: fix or feature that will cause current functionality to change.
```

Comment:
@awentzel can you provide context as to why we aren't enumerating "chore"?

Reply:
@Falkicon is in a previous PR.

Reply:
@awentzel ok - do you have a link? I'm asking for context instead of simply stating that I think chore should be enumerating - I'm assuming I missed something.

Reply:
Updated per collaboration effort